compiler_opt/es/blackbox_optimizers.py

```diff
@@ -1063,7 +1113,9 @@ def get_state(self):
     else:
       return []
 
-  def update_state(self, evaluation_stats):
+  def update_state(
+      self, evaluation_stats: Union[Sequence[float],
```

Comment:
you could define a type at the top, like

`SequenceOfFloats = Union[Sequence[float], npt.NDArray[np.float32]]`

and then use that (less verbose)

Reply:
This change has been applied